test-cypress/integration/import/import.server.files.spec.js

```diff
@@ -1,137 +1,72 @@
+    const baseURI = 'http://purl.org/dc/elements/1.1/';
+    const context = 'http://example.org/context';
+    const messageSuccess = 'Imported successfully';
+    const fileToImport = 'italian_public_schools_links.nt.gz';
+    const blankNodeFile = 'bnodes.ttl';
 
-    after(function () {
-        cy.navigateToPage('Setup', 'Repositories');
-        cy.deleteRepo(existingRepo).wait(1000);
+    beforeEach(() => {
+        repositoryId = 'server-import-' + Date.now();
+        cy.createRepository({id: repositoryId});
+        ImportSteps.visitServerImport(repositoryId);
     });
 
-    it('Test import Server files successfully without changing settings', function () {
-        cy.chooseServerFilesTab();
-        cy.selectDistinctFileToImport(fileToImport);
-        cy.importChangingSettings().wait(2000);
-        cy.verifyMessageOccurs(messageSuccess);
+    afterEach(() => {
+        cy.deleteRepository(repositoryId);
     });
 
-    it('Test import Server files successfully with changing base uri', function () {
-        cy.chooseServerFilesTab();
-        cy.selectDistinctFileToImport(fileToImport);
-        cy.importChangingSettings(true);
-        cy.fillBaseURIInputField(baseURI).wait(1000);
-        cy.clickImportBtnOnPopUpMenu().wait(1500);
-        cy.verifyMessageOccurs(messageSuccess);
-        cy.confirmTextPresentsInDistinctFileInfo(baseURI, fileToImport);
+    it('Test import Server files successfully without changing settings', () => {
+        ImportSteps
+            .selectServerFile(fileToImport)
+            .importServerFiles()
+            .verifyImportStatus(fileToImport, messageSuccess)
+            .verifyImportStatusDetails(fileToImport, '"preserveBNodeIds": false,');
     });
 
-    it('Test import Server files successfully with changing context on newly created repository', function () {
-        let repoId = 'repo1' + Date.now();
-        // First navigate to create repository
-        cy.navigateToPage('Setup', 'Repositories');
-        // Create new one
-        cy.createNewRepo(repoId).wait(2000);
-        cy.selectRepo(repoId).wait(1000);
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-
-        cy.chooseServerFilesTab();
-        cy.selectDistinctFileToImport(fileToImport);
-        cy.importChangingSettings(true);
-        cy.selectNamedGraphCheck();
-        cy.fillNamedGraphInputField(context).wait(1000);
-        cy.clickImportBtnOnPopUpMenu().wait(2000);
-        cy.verifyMessageOccurs(messageSuccess);
-        cy.confirmTextPresentsInDistinctFileInfo(context, fileToImport);
-
-        // First navigate to create repository
-        cy.navigateToPage('Setup', 'Repositories');
-        cy.deleteRepo(repoId).wait(1000);
-        cy.selectRepo(existingRepo).wait(1000);
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-        cy.chooseServerFilesTab();
+    it('Test import Server files successfully with changing settings', () => {
```

Comment:
I've combined several tests in this one cause there is no need to test each setting separately when we are not testing how the data is actually imported (it's tested what is shown in the popup).

Reply:
Yes, in general I agree that we should test all settings in one test.